docs/_posts/examples/3400-01-04-center-on-marker.html

```diff
@@ -0,0 +1,95 @@
+---
+layout: example
+category: example
+title: Center on marker
+description: Using featuresAt and flyTo to center on a marker
+---
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v8',
+    center: [-90.96, -0.47],
+    zoom: 8
+});
+
+map.on('style.load', function () {
+    map.addSource("myMarkers", {
+      "type": "geojson",
+      "data": {
+        "type": "FeatureCollection",
+        "features": [
+          {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -91.395263671875,
+                -0.9145729757782163
+
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -90.32958984375,
+                -0.6344474832838974
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -91.34033203125,
+                0.01647949196029245
+              ]
+            }
+          }
+        ]
+      }
+    });
+
+    map.addLayer({
+        "id": "myMarkers",
+        "interactive": true,
+        "type": "symbol",
+        "source": "myMarkers",
+        "layout": {
+          "icon-image": {
+            "base": 1,
+            "stops": [
+                [
+                    1,
+                    "star-15"
+                ],
+                [
+                    17,
+                    "star-15"
+                ]
+            ]
+          },
+        },
+        "paint": {}
+    });
+});
+
+map.on('click', function(e){
+  map.featuresAt(e.point, { radius: 20, includeGeometry: true }, function(err, features){
```

Comment:
```
    map.featuresAt(e.point, {layer: 'myMarkers', radius: 20, includeGeometry: true}, function (err, features) {
        if (err) throw err;
        if (features.length) {
            map.flyTo({center: features[0].geometry.coordinates});
        }
    });
```

* Uses the `layer` option for `featuresAt` to avoid getting results for any other layers
* Uses the coordinates from the marker itself, like the Mapbox.js example

Reference #1002 for a future API enhancement that would simplify this example to:

```
map.on('click', 'myMarkers', function (e) {
    map.flyTo({center: e.features[0].geometry.coordinates});
});
```